### PR TITLE
Nikhil_Assignment_35_Done_10Min

### PR DESCRIPTION
**Pull request by NikhilMJun20**
-Completed and pushed Assignment # 35
-Please review and comment.

-PFB screenshot of output of the program:-

Annotation 2020-07-28 152942

### CODE REVIEW

**Review by VaishnaviVJun20 — APPROVED** [review]
--Code looks good to me
-- logic for second approach looks perfect

**Comment by NikhilMJun20** [comment]
> --Code looks good to me
> -- logic for second approach looks perfect

Thank you, @VaishnaviVJun20 !

File: src/nikhilMJun20/UnionOfArrayLists.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+package nikhilMJun20;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.ListIterator;
+
+public class UnionOfArrayLists {
+	ArrayList<String> createUnion(ArrayList<String> arrList1, ArrayList<String> arrList2) {
+		//Below method will not work correctly if duplicate entries exist
+		arrList1.removeAll(arrList2);
```
Comment (KrishnaKTechnocredits) [inline src/nikhilMJun20/UnionOfArrayLists.java:18]: if you want to print exact order as given in question then you should,
arrList2.removeAll(arrList1);
arrList1.addAll(arrList2);

print -> arrList1

**Review by KrishnaKTechnocredits — APPROVED** [review]
Code looks good to me, please take a look at comments.

**Pull request merged**